### ISSUE TITLE
Dedicated timeline background color for own messages

### ISSUE THREAD

**Issue by spantaleev** [Nheko-Reborn/nheko#1151]
### The Problem

I'm in a room with 5-6 people and have enabled the **message bubbles** feature.

Each message is in its own bubble which has a background color.

The problem is that one other person and my own messages have the same exact color (`#dfdfd2`).

**Unless your window is very wide** (so that foreign messages will be on the left and your own will be on the right), **it's hard to differentiate my own messages from other people's messages**.

### The Solution

Use a dedicated color for your own messages.

I'm not sure how background colors are currently picked.

I'm guessing there is a list of 5-6 colors and users are assigned one of those based on some logic. If so, having a dedicated one for the current user should be easy.

If colors are computed randomly, it may be harder to pick a dedicated color for your own messages (one which won't be stumbled upon by computing colors for the other users).

A good background color for your own messages might be `#dbedf9` (light blue) or `#b6cded` (darker blue), which are part of the official logo icon.

### Happens in the latest version

- [X] Yes, this feature is still missing.

**Comment by deepbluev7** [comment]
> I'm guessing there is a list of 5-6 colors and users are assigned one of those based on some logic. If so, having a dedicated one for the current user should be easy.

No, there are 256 or however many steps there is for the hue parameter. Usually every user has a unique color.

So the only color that would be left over for your own bubbles would be shades of grey, which is kinda boring.

**Comment by spantaleev** [comment]
It's strange that a room with 6 users only was enough to yield a color collision (`#dfdfd2`).

There should be a way to reserve some of that color space for your own messages. If others collide, it's not ideal, but it's still better than having your own message collide with others.

**Comment by spantaleev** [comment]
![Screenshot_20220812_111124](https://user-images.githubusercontent.com/388669/184314673-b0d45f13-ad4d-4610-bb88-6dc6e3af8a60.png)

Here's what the problem looks like visually.

The window width (timeline width) is low enough, so that my own message is not pushed to the right too much. It also has no user avatar.

This makes it look like a sequence of messages by **Another** (look at the sequence of 2 messages at the bottom). The only difference is that my message is slightly pushed to the right. How much it's pushed depends on various things, but.. I think it's not enough.

---

I think that it can be solved in multiple ways:

- dedicated background color for own messages
- showing own user avatar next to own messages. What's the reasoning for hiding it? Showing it is a good way to differentiate messages even further (which will help people with various accessibility problems as well). Most people don't get tired of seeing their own face. `element-web` with message bubbles enabled shows your avatar (without the name) on top of every *sequence* of messages. Other people's bubbles in `element-web` have both the avatar and the name.
- right-alignment (we already have it)

**Comment by deepbluev7** [comment]
Avatars are mostly hidden to save space (including your username) on mobile, but I also often miss it, so maybe that should just be an option.
